### ISSUE TITLE
[BUG] - TraceBlockFetchClient doesn't work

### ISSUE THREAD

**Issue by karknu** [input-output-hk/cardano-node#1584]
**Interna**

**Summary**
`TraceBlockFetchClient` doesn't work.

**Steps to reproduce**
1. Enable the tracer by adding `"TraceBlockFetchClient": true,` to the config file.
2. Run the node
3. grep for "BlockFetchClient"
4. Notice the lack of messages.

**Expected behavior**
Enabling TraceBlockFetchClient should result in the logging of messages related to the block fetch client.

**System info (please complete the following information):**
- Node version 1.18.0

**Comment by CodiePP** [comment]
these messages are traced using `Debug` severity, so `minSeverity` filter must be set to Debug.

**Comment by karknu** [comment]
https://github.com/input-output-hk/cardano-node/blob/7d68626f1f9a56ab318fda7be4cb43122b31dad6/cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs#L111
To me it looks like they should be traced at a `minSeverity` of `Info`, but someone is overwriting it.

**Comment by vix-io** [comment]
hi @karknu please can you check if this is still relevant?

**Comment by karknu** [comment]
Yes, the tracer is still broken.

**Comment by dcoutts** [comment]
```
diff --git a/cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs b/cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs
index 518eda7c..7006e430 100644
--- a/cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs
+++ b/cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs
@@ -85,7 +85,8 @@ instance HasSeverityAnnotation (TraceSendRecv a) where
 
 
 instance HasPrivacyAnnotation a => HasPrivacyAnnotation (TraceLabelPeer peer a)
-instance HasSeverityAnnotation a => HasSeverityAnnotation (TraceLabelPeer peer a)
+instance HasSeverityAnnotation a => HasSeverityAnnotation (TraceLabelPeer peer a) where
+  getSeverityAnnotation (TraceLabelPeer _p a) = getSeverityAnnotation a
```

**Comment by coot** [comment]
Fixed in 6157e1c3f0ee9484f6cfce390b81ed97dd4c0025
